lib/uniform/blueprint.ex

```diff
@@ -382,7 +382,14 @@ defmodule Uniform.Blueprint do
             (file :: String.t(), Uniform.App.t() -> String.t())
             | (file :: String.t() -> String.t())
         ) :: term
-  defmacro modify(path_or_regex, {:fn, _, _} = fun) do
+  defmacro modify(path_or_regex, {:&, _, [{:/, _, _}]} = fun) do
+    quote do
+      Uniform.Blueprint.validate_path_or_regex!(unquote(path_or_regex))
+      Module.put_attribute(__MODULE__, :modifiers, {unquote(path_or_regex), unquote(fun)})
+    end
+  end
+
+  defmacro modify(path_or_regex, {call, _, _} = fun) when call in [:fn, :&] do
```

Comment:
This line is the main change which treats `& &1` identically to `fn x -> x end`. Note `when call in [:fn, :&]`